1.10/installing/custom/system-requirements/install-docker-centos.md

```diff
@@ -20,6 +20,13 @@ In addition to the general [Docker requirements and recommendations for DC/OS][1
   mkfs -t xfs -n ftype=1 /dev/sdc1
   ```
 
+* If you are using Docker 1.12 or Docker 1.13, you must add `MountFlags=private` in `/etc/systemd/system/docker.service.d/execstart.conf` and then restart the Docker daemon with this command:
```

Comment:
@jieyu - our soak testing was successful only after MountFlags= private.  (and that's reason we are having this in release notes).

And yes, in our soak, I believe, we test with persistent volume support too. 

The persistent volume itself was tested for the dcos-image (but that is without this flag).